subsys/mpsl/fem/Kconfig

```diff
@@ -61,15 +61,34 @@ config MPSL_FEM_SIMPLE_GPIO
 
 endchoice	# MPSL_FEM_CHOICE
 
-if MPSL_FEM_NRF21540_GPIO
+if MPSL_FEM_NRF21540_GPIO && MPSL_FEM
 
 config MPSL_FEM_NRF21540_TX_GAIN_DB
 	int "TX gain of the nRF21540 PA amplifier in dB"
+	default MPSL_FEM_NRF21540_TX_GAIN_DB_POUTB
+	help
+	  With the GPIO implementation, the Kconfig can be set either to TX gain of POUTA
+	  or TX gain of POUTB. Using an unsupported value results in build assertion.
+
+config MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA
+	int "TX gain value (POUTA)"
 	default 20
 	help
-	  The default value of 20 dB is based on nRF21540 Product Specification
-	  (v1.0) and it corresponds to the configuration in which the pin
-	  MODE=0 and pin POUTA_SEL=0
+	  The default value of 20 dBm is based on nRF21540 Product Specification (v1.0) and it
```

Comment:
Sorry to interfere, as I was not assigned to review. Came here as DTM also has a parallel FEM support. Just a nitpick from me: I think the unit should be dB instead of dBm (as it is relative gain and not absolute output power value).